app/package.json

```diff
@@ -73,13 +75,13 @@
   },
   "scripts": {
     "prebuild": "rimraf ../src/phoenix/server/static",
```

Comment:
pre-build doesn't work with pnpm so you have to do it explicitly on 78 via &

Reply:
resolved in 189f3d1151ec86fba530e4fe8b13715fe99e1eb2